src/core/config/border_routing.h

```diff
@@ -76,4 +76,22 @@
 #define OPENTHREAD_CONFIG_BORDER_ROUTING_MAX_ON_MESH_PREFIXES 16
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_BORDER_ROUTING_ROUTER_ACTIVE_CHECK_TIMEOUT
+ *
+ * Specifies the timeout in msec for a discovered router on infra link side.
+ *
+ * This parameter is related to mechanism to check that a discovered router is still active.
+ *
+ * After this timeout elapses since the last received message (a Router or Neighbor Advertisement) from the router,
+ * routing manager will start sending Neighbor Solidification (NS) probes to the router to check that it is still
+ * active.
+ *
+ * This parameter can be considered to large value to practically disable this behavior.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_BORDER_ROUTING_ROUTER_ACTIVE_CHECK_TIMEOUT
+#define OPENTHREAD_CONFIG_BORDER_ROUTING_ROUTER_ACTIVE_CHECK_TIMEOUT (120 * 1000) // (in msec).
```

Comment:
nit: current draft spec proposes 60 seconds
```suggestion
#define OPENTHREAD_CONFIG_BORDER_ROUTING_ROUTER_ACTIVE_CHECK_TIMEOUT (60 * 1000) // (in msec).
```

Reply:
Changed it in the new push.